server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4612,6 +4626,55 @@ public void finalizeStop(VirtualMachineProfile profile, Answer answer) {
         return vmParamPair;
     }
 
+    private Pod getDestinationPod(Long podId, boolean isRootAdmin) {
+
```

Comment:
Remove extra new line, and same for other related methods.

Reply:
done.